common/lib/capa/capa/capa_problem.py

```diff
@@ -978,7 +974,7 @@ def response_a11y_data(self, response, inputfields, responsetype_id, problem_dat
             for description in description_tags:
                 descriptions[
                     "description_%s_%i" % (responsetype_id, description_id)
-                ] = description.text
+                ] = HTML(stringify_children(description))
```

Comment:
@robrap Until now, as you know, course authors are part of a mutually trusted group of users who can manipulate course content that is ultimately executed in our users' browsers.  However, just as with any well-intentioned developer, course authors may appreciate having automated tools in place that would prevent/detect any inadvertent (security) issues.  Additionally, course authors may not be aware of the details of where their data is being stored (for example, as `aria-label` HTML attributes in these cases ?).

So I would suggest carefully considering how/where user-provided data is stored/escaped, whether it be from a "trusted" user-base or not.

(Not sure if this was the feedback you were asking for..)